### PR TITLE
Add integration test in CI

### PR DESCRIPTION
**Pull request by amrita-shrestha**
This PR adds integration test in CI

## Related issue 
- https://github.com/owncloud/ocis-php-sdk/issues/16

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -56,10 +61,163 @@ def main(ctx):
...
+    ]
+
+def postgresService():
+    return [
+        {
+            "name": "postgres",
+            "image": POSTGRES_ALPINE,
+            "environment": {
+                "POSTGRES_DB": "keycloak",
+                "POSTGRES_USER": "keycloak",  # needed for checking config later
```
Comment (saw-jan) [inline .drone.star:177]:
```suggestion
                "POSTGRES_USER": "keycloak",
```

File: tests/integration/Owncloud/OcisSdkPhp/OcisTest.php [hunk 2]
```diff
@@ -22,6 +22,7 @@ class OcisTest extends TestCase
 
     public function setUp(): void
     {
+        $this->ocisUrl = (getenv('OCIS_URL') !== false) ? getenv('OCIS_URL') : 'https://ocis.owncloud.test';
```
Comment (saw-jan) [inline tests/integration/Owncloud/OcisSdkPhp/OcisTest.php:25]: if it works
```suggestion
        $this->ocisUrl = getenv('OCIS_URL') || 'https://ocis.owncloud.test';
```

File: .drone.star [hunk 3]
```diff
@@ -56,10 +61,163 @@ def main(ctx):
...
+
+def buildOcis():
+    ocis_repo_url = "https://github.com/owncloud/ocis.git"
+    return [
+        {
+            "name": "clone-ocis",
+            "image": OC_CI_GOLANG,
+            "commands": [
+                "source .drone.env",
+                "git clone -b $OCIS_BRANCH --single-branch %s" % ocis_repo_url,
```
Comment (saw-jan) [inline .drone.star:138]: pull only the head commit
```suggestion
                "git clone -b $OCIS_BRANCH --single-branch --depth 1 %s" % ocis_repo_url,
```

File: tests/integration/Owncloud/OcisSdkPhp/OcisTest.php (hunk 2 above)
Comment (saw-jan) [reply tests/integration/Owncloud/OcisSdkPhp/OcisTest.php:25]: php short circuit doesn't work as expected

Comment (saw-jan) [inline tests/integration/Owncloud/OcisSdkPhp/OcisTest.php:25]: short form
```suggestion
        $this->ocisUrl = getenv('OCIS_URL') ?: 'https://ocis.owncloud.test';
```

**Review by individual-it — APPROVED** [review]
LGTM Well done!

In the next step we should use caches for ocis, can be part of https://github.com/owncloud/ocis-php-sdk/issues/53

**Pull request merged**